src/tests/unit/services/SessionEventProcessor.test.ts

```diff
@@ -4,28 +4,31 @@ import { Logger } from "@aws-lambda-powertools/logger";
 import { SessionEventProcessor } from "../../../services/SessionEventProcessor";
 import { mock } from "jest-mock-extended";
 import { DynamoDBStreamEvent } from "aws-lambda";
-import { VALID_DYNAMODB_STREAM_EVENT } from "../data/dynamodb-stream-record";
+import { VALID_DYNAMODB_STREAM_EVENT, VALID_DYNAMODB_STREAM_EVENT_WITH_PO_DETAILS } from "../data/dynamodb-stream-record";
 import { IPRService } from "../../../services/IPRService";
 const { unmarshall } = require("@aws-sdk/util-dynamodb");
 
 let sessionEventProcessorTest: SessionEventProcessor;
 const mockIprService = mock<IPRService>();
-const logger = mock<Logger>();
+const mockLogger = mock<Logger>();
 const metrics = new Metrics({ namespace: "F2F" });
 let streamEvent: DynamoDBStreamEvent;
+let streamEventWithPoDetails: DynamoDBStreamEvent;
 jest.spyOn(console, "log").mockImplementation(() => {});
 
 describe("SessionEventProcessor", () => {
 	beforeAll(() => {
-		sessionEventProcessorTest = new SessionEventProcessor(logger, metrics);
+		sessionEventProcessorTest = new SessionEventProcessor(mockLogger, metrics);
 		// @ts-ignore
 		sessionEventProcessorTest.iprService = mockIprService;
 		streamEvent = VALID_DYNAMODB_STREAM_EVENT;
+		streamEventWithPoDetails = VALID_DYNAMODB_STREAM_EVENT_WITH_PO_DETAILS;
 	});
 
 	beforeEach(() => {
 		jest.clearAllMocks();
 		streamEvent = VALID_DYNAMODB_STREAM_EVENT;
+		streamEventWithPoDetails = VALID_DYNAMODB_STREAM_EVENT_WITH_PO_DETAILS;
 	});
 
 	it("When all the necessary fields are populated in the session Event record, sends email and updates notified flag", async () => {
```

Comment:
[nit] shall we update the it to include `old`?

Reply:
Didnt get this comment, can you elaborate please?

Reply:
Sorry, can we update the `it` block to reflect the change that you've made in the expectation.

Something like 
`it("When all the necessary fields are populated in the session Event record, sends old email and updates notified flag"...`